tests/integration/block_create_test.go

```diff
@@ -56,7 +56,7 @@ func (s *BlockCreateSuite) SetupSuite() {
 	s.namespace = "block-k8s-ns"
 	mons := 1
 	rbdMirrorWorkers := 1
-	s.op, s.kh = StartTestCluster(s.T, s.namespace, "bluestore", false, false, mons, rbdMirrorWorkers, installer.VersionMaster, installer.MimicVersion)
```

Comment:
nit: for readability I'd suggest following the pattern of defining the variable `enableCSI`, then pass to the method, similarly to most other params.